buyer-api/src/blockchain/contractEventSubscribers/batchPaymentsSubscribers.js

```diff
@@ -28,13 +28,23 @@ export const updateBuyerStats = async (
   );
 };
 
+/**
+ * @function storeLockingKeyHashByPayIndex Is triggered when
+ * the payment is registered,
+ * storing locally a relationship between the pay index and
+ * the payment transaction hash
+ * @typedef PaymentRegisteredEventValues
+ * @property {number} payIndex Pay index
+ * @param {UnlockEventValues} eventValues The values emmited by the BatPay PaymentRegistered event
+ */
 export const storeLockingKeyHashByPayIndex = async ({ payIndex }, { transactionHash }) =>
```

Comment:
Update docs with two parameters, and there's no declaration of type `UnlockEventValues`